Sources/GRPCReflectionService/Server/ReflectionService.swift

```diff
@@ -207,6 +211,27 @@ internal struct ReflectionServiceData: Sendable {
 internal final class ReflectionServiceProvider: Grpc_Reflection_V1_ServerReflectionAsyncProvider {
   private let protoRegistry: ReflectionServiceData
 
+  internal init(protoFilePaths: [String]) throws {
+
+    let binaryFilesURLs = protoFilePaths.map {
+      #if os(Linux)
+      URL(fileURLWithPath: $0)
+      #else
+      if #available(macOS 13.0, iOS 16.0, tvOS 16.0, watchOS 9.0, *) {
+        URL(filePath: $0, directoryHint: .notDirectory)
+      } else {
+        URL(fileURLWithPath: $0)
+      }
+      #endif
+    }
+    let binaryData = binaryFilesURLs.map { Data(base64Encoded: try! Data(contentsOf: $0)) }
+    let fileDescriptorProtos = binaryData.map {
+      try! Google_Protobuf_FileDescriptorProto(serializedData: $0!)
+    }
```

Comment:
We can't `try!` these, they may legitimately fail and this would cause a crash.

We should just do the basic thing:

```swift
var fileDescriptorProtos = [Google_Protobuf_FileDescriptorProto]()
for path in paths {
  // ...  
  fileDescriptorProtos.append(...)
}

```